core.py

```diff
@@ -1,11 +1,8 @@
 import logging
-import os
-import sys
+import os, sys
 import threading
 
-import sqlalchemy
-import sqlalchemy.ext.declarative as sed
-import sqlalchemy.orm
+import sqlalchemy, sqlalchemy.ext.declarative as sed
```

Comment:
Can you split this into multiple lines?

```suggestion
import sqlalchemy
import sqlalchemy.ext.declarative as sed
```

It's just a matter of personal preference, it should work anyways :)